mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -486,6 +484,8 @@ public void onStopTrackingTouch(SeekBar seekBar) {
    		break;
     	case TYPE_CHART:
     		MySmartImageView chartImage = (MySmartImageView)widgetView.findViewById(R.id.chartimage);
+    		//Always clear the drawable so no images from recycled views appear
+    		chartImage.setImageDrawable(null);
```

Comment:
This seems reasonable, but the PR contains other changes not related to the issue.  Can you submit only this change?  We will hopefully have the multidex supported merged soon.  Also please sign your commit message according to https://github.com/openhab/openhab-distro/blob/master/CONTRIBUTING.md .  Thanks!